### PR TITLE
Display distribution on swupd info

### PR DESCRIPTION
**Pull request by litcoder**
Display distribution information on swupd info which is from PRETTY_NAME filed
in os-release file to support distinguishable version information display
in down streams.

Signed-off-by: Brandon Hong <brandon.hong@intel.com>

### CODE REVIEW

File: src/info.c [hunk 1]
```diff
@@ -25,13 +25,16 @@
 enum swupd_code print_update_conf_info()
 {
 	enum swupd_code ret = SWUPD_OK;
-
+	char dist_string[LINE_MAX];
 	int current_version = get_current_version(path_prefix);
-	if (current_version < 0) {
+	bool dist_string_found = get_distribution_string(path_prefix, dist_string);
+
+	if (current_version < 0 || !dist_string_found) {
```
Comment (otaviobp) [inline src/info.c]: If !dist_string_found you have the information about the version to display. So instead you can print

    Distribution: unknown
    Installed version: 12345

And if current_version < 0

    Error: Unable to determine current OS version
    Distribution: Clear Linux OS
    Installed version: unknown

**Comment by otaviobp** [comment]
There was a patch that was merged that may help you when running tests too. I recommend rebasing it again

**Comment by litcoder** [comment]
I updated some of test cases to reflect the new swupd info output however I'm still seeing some failures in usa-config-files.bats when I run local testing. Since those errors were still observed without the patch, I'm thinking this might not directly related to my patch.

**Comment by litcoder** [comment]
After the rebase, most of the issues gone but still have a issue with the test case USA010 and this still fails without the patch in my local testing.

**Comment by otaviobp** [comment]
merged. thanks

**Pull request closed without merging**